draft-ietf-httpbis-digest-headers.md

```diff
@@ -0,0 +1,893 @@
+---
+title: Resource Digests for HTTP
+abbrev: RDHTTP
+docname: draft-polli-resource-digests-http-latest
+category: std
+
+ipr: trust200902
+area: General
+workgroup:
+keyword: Internet-Draft
```

Comment:
remove

Reply:
should I put something else here, eg. "Digest" ?

Reply:
Sure.